README.md

```diff
@@ -1,70 +1,82 @@
-# Getting Started with Create React App
+![default-monochrome](https://user-images.githubusercontent.com/56429354/205527587-ac4a2eee-a331-4493-ab17-744ae465e646.svg#gh-light-mode-only)
+![default](https://user-images.githubusercontent.com/56429354/205528723-bced613e-619a-4c0f-aa30-e4c53ce7e501.svg#gh-dark-mode-only)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+>Book car is a platform to book a car that is present in different cities for a given period of time. The user selects the duration of rental per car model type and price range. if a user is an admin he can manage the car resources by updating the list of cars in the database.
 
-## Available Scripts
+## Backend Repository
+[GitHub Link](https://github.com/Gedewon/CarBooking-Backend.git)
```

Comment:
- [ ] I couldn't find any open PR in the backend repository to review.  For your next review request, kindly open one that points from dev to main, so a proper review for the backend can be performed. 👍